src/WebJobs.Script/Binding/Http/ProxyHttpExtensions.cs

```diff
@@ -0,0 +1,391 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    internal static class ProxyHttpExtensions
```

Comment:
These are also being used by .Net and are also being used by Gibraltar. I removed the the Extension methods and made them as regular static only based on a previous comment.